app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumNewPostActivityTest.kt

```diff
@@ -55,7 +60,41 @@ class ForumNewPostActivityTest {
         }.join() // Adding a timeout is not possible on Kotlin for the same silly reasons
     }
 
+    @Test
+    fun notifiedWhenPostReplyWorks() {
+        // Create new post with notifications on
+        onView(withId(R.id.newPostCategoryDropdown))
+            .perform(replaceText(CATEGORY_TEST_STRING))
+        onView(withId(R.id.newPostTitleEditTxt))
+            .perform(replaceText(QUESTION))
+        onView(withId(R.id.switch_enable_notifications))
```

Comment:
I see that we should have been more consistent in the ids naming by settling on either snake or camel case.

Reply:
Not dramatic though